lib/public/views/DataPasses/ActiveColumns/dataPassesActiveColumns.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { textFilter } from '../../../components/Filters/common/filters/textFilter.js';
+
+/**
+ * List of active columns for a generic data passes table
+ */
+export const dataPassesActiveColumns = {
+    name: {
+        name: 'Name',
+        visible: true,
+        format: (name) => h('.flex-row.flex-wrap', [
+            h('.ph1.mh1.w-50.text-ellipsis', name),
```

Comment:
https://github.com/AliceO2Group/Bookkeeping/pull/1331#discussion_r1457417345

Reply:
How does this reply to my question?

Reply:
it seems I copied a bad link, here is correct one (https://github.com/AliceO2Group/Bookkeeping/pull/1331#discussion_r1457418345), I put link because I see this as repetition of the other question